### PR TITLE
UT Committee scraper

### PR DESCRIPTION
**Pull request by EhtishamSabir**
It's working UT Committee scraper waiting for review.

### CODE REVIEW

**Review by johnseekins — APPROVED** [review]

**Review by chrisyamas — CHANGES_REQUESTED** [review]
Once those changes have been implemented, this PR should be ready for merge!

File: scrapers_next/ut/committees.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            )
+            # extracting members
+            members_list = each_committee["members"]
+            for each_memeber in members_list:
+                member_name_split = each_memeber["name"].split(",")
+                member_name_split.reverse()
+                member_name = " ".join(member_name_split).replace(".", "").strip()
+
+                role = each_memeber["position"]
+                com.add_member(name=member_name, role=role)
```
[the remark this answers is not in the document]
Comment (EhtishamSabir) [reply scrapers_next/ut/committees.py:49]: I applied your suggestions... but still sometimes middle name only contains first character like...

Candice B. Pierucci

**Comment by EhtishamSabir** [comment]
poetry run black scrapers_next/ut/committees.py
All done! ✨ 🍰 ✨
1 file left unchanged.


I'm doing it still facing this lint error not sure why

**Comment by johnseekins** [comment]
If you want to pull `main` into your branch, it should start linting correctly. There was a mistake in the CI that was using an inconsistent version of the linter.

File: scrapers_next/ut/committees.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    source = URL(
+        "https://le.utah.gov/data/committees.json",
+        timeout=10,
+    )
+
+    def process_page(self):
+        legislators_url = "https://le.utah.gov/data/legislators.json"
+        membership = {}
+        response = requests.get(legislators_url)
+        for each_legislator in response.json()["legislators"]:
```
Comment (johnseekins) [inline scrapers_next/ut/committees.py:22]: We call `response.json()` frequently. Any reason we can't call it _once_ before these loops and work on the output instead?

Comment (EhtishamSabir) [reply scrapers_next/ut/committees.py:22]: Actually 1 time is from self & other is from requests I was trying different ways I thought I had lint issue Let me revert.

**Comment by EhtishamSabir** [comment]
again Lint issue

File: scrapers_next/ut/committees.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+                member_id = each_member["id"]
+                name = membership[member_id]
+                role = each_member["position"]
+                com.add_member(name, role=role.title())
+
+            if not com.members:
+                logging.warning(f"No membership data found for: {name}")
+                continue
+            if len(each_committee.get("link")):
+                com.add_link(each_committee["link"], note="Committee web page")
```
Comment (chrisyamas) [inline scrapers_next/ut/committees.py:70]: Change note to be `note="homepage"`.

**Review by chrisyamas — APPROVED** [review]
Looks good, we're passing lint checks now, and data is solid. Thanks for your patience on this one!

**Pull request merged**